back/engines/free/email_campaigns/README.md

```diff
@@ -65,11 +65,11 @@ module EmailCampaigns
 end
 ```
 
-### `before_send`
+### `filter`
 
-Registered `before_send` actions are possibly passed `time` and `activity` options, depending on how the campaign is  triggered - through a schedule or through an activity trigger.
+Registered `filter` actions are possibly passed `time` and `activity` options, depending on how the campaign is  triggered - through a schedule or through an activity trigger.
 
-The `before_send` actions have 2 purposes:
+The `filter` actions have 2 purposes:
 * Run any code that would be needed before send a campaign
```

Comment:
Because this is maybe not just filtering (according to this comment) maybe a better term would be `filter_and_prepare`? Not a show stopper though